lib/trento/application/projectors/host_projector.ex

```diff
@@ -81,6 +82,20 @@ defmodule Trento.HostProjector do
     end
   )
 
+  project(
+    %HostRemovedFromCluster{
+      host_id: id
+    },
+    fn multi ->
+      query =
+        Ecto.Query.from(h in HostReadModel,
```

Comment:
Yes but no, I mean we don't know the order of the events, and we want to make sure to not update anything that is already soft delete, to mantain consistency and logic.

This guard will just make that happen. Right know I know that this could potentially be a no-op, but I don't really like the idea to pollute deleted entities :D 

Anyway if you like it we can merge.

Reply:
I don't really think this could ever happen. You cannot de-register a host without de-registering it first from a cluster (and in any case, what you "say" about polluting would be reflecting the source of truth in that strange scenario XD)
I guess that the cost of doing an extra query is low.
Anyway, you already had my green light hehe

Reply:
@arbulu89 @CDimonaco I think it's my fault on this one.
We cannot guarantee the order of the events, since HostRemovedFromCluster is emitted by the Cluster aggregate and might arrive after the HostDeregistered event which is emitted concurrently by the Host aggregate. I was confused and thought we were deleting the entry for good, therefore my suggestion was to use `update_all`.

I still prefer Xabi's solution since the host is just soft deleted so the read model can be found. It doesn't matter if we update a soft deleted read model, actually, we must update it to have a consistent entry if we then plan to roll back the deregistration.